client/question/services/FeedbackGeneratorService.js

```diff
@@ -268,28 +360,45 @@ tie.factory('FeedbackGeneratorService', [
      * @private
      */
     var _getCorrectnessTestFeedback = function(
-      outputFunctionName, testCase, observedOutput) {
+      outputFunctionName, testCase, observedOutput, testSuiteId,
+      testSuiteIndex) {
```

Comment:
Please update the docstring when adding new args.

Reply:
Done